### PR TITLE
update readme.md #9 #hacktoberfest

### PR DESCRIPTION
**Pull request by pmichielsen**
My first one 😄 #hacktoberfest

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ PayPay statement can be acquired here: https://business.paypal.com/merchantdata/
 
 Zenmoney csv import interface is here: https://zenmoney.ru/a/#import
 
-Place the file called Download.csv near the script, then run:
+Place the file called Download.csv in the same directory as the script, then run from the command line:
```
Comment (goblinqueen) [inline README.md]: This is no longer valid as the location of input file is now provided by -f

File: README.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ PayPay statement can be acquired here: https://business.paypal.com/merchantdata/
...
 
-    $ python3 zenpal.py > zenpal.csv
\ No newline at end of file
+    $ python3 zenpal.py [-h] -f FILE [-o OUTPUT_FILE] [-a APPEND]
+
```
Comment (goblinqueen) [inline README.md:16]: please add an example that can be copied and pasted into console as is (preferably with an explanation what it does). It doesn't need to show all functions, just act as a quickstart.
No need to copy-paste the script's --help  -- a readme is not a full documentation, more of a overview/quickstart

File: README.md (hunk 1 above)
Comment (pmichielsen) [reply README.md]: Ah true! I will amend

**Review by goblinqueen — CHANGES_REQUESTED** [review]
Thanks a lot! I left some comments, please update :)

File: README.md (hunk 2 above)
Comment (pmichielsen) [reply README.md:16]: Acknowledged. Though with the size of the readme it can do no harm

**Review by goblinqueen — APPROVED** [review]

**Comment by goblinqueen** [comment]
Thanks, that helps a lot!
There are some minor tweaks, but not worth requesting changes, I'll do it myself some day.

**Comment by pmichielsen** [comment]
Cheers. Just trying getting to grips with contributing to public repositories

**Pull request merged**